### PR TITLE
Queue support without cancel

### PR DESCRIPTION
**Pull request by PostsDesert**
I have not found any good way to implement this where Alexa will reply with a statement. If the code sends a statement and `@ask.on_playback_nearly_finished()` has already fired (which is pretty close to the beginning of the song), the song will not be queued. Also Alexa only accepts one response per requests so if I try to enqueue it the speech will not be spoken until in between the songs (which could be very annoying if you queue multiple songs). 

An Alexa Card combined with a `"UndoQueueIntent"` may be the best approach since it lets the user know via a Simple Card and the user could cancel from there. Alternatively you could have a repromt for confirmation but I feel like this would be too wordy since most user are not going to want to "pause" the music for a long period of time while queuing a song.  Cheers  🎉.

### CODE REVIEW

File: geemusic/intents/selection.py [hunk 1]
```diff
@@ -158,3 +158,26 @@ def play_artist_radio(artist_name):
...
+        return statement("You must first play a song")
+
+    # Fetch the song
+    song = api.get_song(song_name, artist_name=artist_name)
+
+    if song is False:
+        return statement("Sorry, I couldn't find that song")
+
+    # Queue the track in the list of song_ids
+    queue.song_ids.append(song['storeId'])
```
Comment (stevenleeg) [inline geemusic/intents/selection.py]: I don't know that this is the best way to do this, as it might cause `queue.current_track()` (see [here](https://github.com/stevenleeg/geemusic/blob/master/geemusic/utils/music_queue.py#L35-L39)) to break, since the track definition won't be in the queue. This would prevent the "what's currently playing?" intent from functioning (and might cause the server to crash).

We might want to add something like `queue.enqueue_track(song)` which handles storing the track definition in addition to adding the ID to the queue.

File: geemusic/intents/selection.py [hunk 2]
```diff
@@ -158,3 +158,26 @@ def play_artist_radio(artist_name):
...
+def queue_song(song_name, artist_name):
+    api = GMusicWrapper.generate_api()
+
+    app.logger.debug("Queuing song %s by %s" % (song_name, artist_name))
+
+    if len(queue.song_ids) == 0:
+        return statement("You must first play a song")
+
+    # Fetch the song
+    song = api.get_song(song_name, artist_name=artist_name)
```
Comment (stevenleeg) [inline geemusic/intents/selection.py]: This works if `artist_name` is `None`, right?

Comment (PostsDesert) [inline geemusic/intents/selection.py]: yep

File: geemusic/intents/selection.py (hunk 1 above)
Comment (PostsDesert) [inline geemusic/intents/selection.py]: Yeah, that would make a lot of sense 🤔. I can't believe I overlooked that 😀.

**Comment by stevenleeg** [comment]
I merged master in and updated this PR to remove the api generation calls. I'm going to close this PR and continue discussion in #40.
